test/test_sendgrid.py

```diff
@@ -144,11 +145,11 @@ def test_hello_world(self):
         content = Content(
             "text/plain", "and easy to do anywhere, even with Python")
         mail = Mail(from_email, subject, to_email, content)
-        self.assertTrue(mail.get() == {
-            'content': [{'type': 'text/plain', 'value': 'and easy to do anywhere, even with Python'}],
-            'personalizations': [{'to': [{'email': 'test@example.com'}]}], 'from': {'email': 'test@example.com'},
-            'subject': 'Sending with SendGrid is Fun'
-        })
+        self.assertTrue(
+            mail.get() == {'content': [{'type': 'text/plain', 'value': 'and easy to do anywhere, even with Python'}],
+                           'personalizations': [
+                               {'to': [{'email': 'test@example.com'}]}], 'from': {'email': 'test@example.com'},
+                           'subject': 'Sending with SendGrid is Fun'})
```

Comment:
It still seems like this should be changed from `self.assertTrue` to `self.assertEqual`? Maybe something like this?

```python
self.assertEqual(mail.get(), {
    'content': [{
        'type': 'text/plain',
        'value': 'and easy to do anywhere, even with Python'
    }],
    'personalizations': [{
        'to': [{'email': 'test@example.com'}]
    }],
    'from': {'email': 'test@example.com'},
    'subject': 'Sending with SendGrid is Fun'
})

```

Reply:
@42B 
You can create a new issue for this. I used the same code from master.